vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/storage/etcd/testing/utils.go

```diff
@@ -169,6 +169,7 @@ func configureTestCluster(t *testing.T, name string, https bool) *EtcdTestServer
 	if err != nil {
 		t.Fatal(err)
 	}
+	m.AuthToken = "simple"
```

Comment:
Yeah it's the token based stuff - this token lets you mutate this subset of the key space, etc.